[NCE-917] Update docs for prepaid cards (#26)
wegift/api-docs: features/prepaid-visa/spend-controls.mdx

> ---

Reviewer: This page leaves an open question of where these template IDs come from. Have we got that yet?

Author: I've left at the bottom of the page some info on how to get that\
<img width="913" height="173" alt="image" src="https://github.com/user-attachments/assets/c6ff7569-e981-49de-9af0-d917c203262f" />

title: "Card spend controls"

> description: "Restrict where a prepaid Visa card can be spent using Runa-authored spend rule templates (Merchant Category Code (MCC) allow/deny lists)."

Reviewer: Description should be very short otherwise overflows where it's used for (share cards &c) and should not end with a full stop. Should be brief overview without repeating too much.

```suggestion
description: Limit which merchants a Prepaid Visa Card can be used for
```
